sdk/schemaregistry/azure-data-schemaregistry-apacheavro/pom.xml

```diff
@@ -15,11 +15,11 @@
   </parent>
 
   <groupId>com.azure</groupId>
-  <artifactId>azure-data-schemaregistry-avro</artifactId>
-  <version>1.0.0-beta.6</version> <!-- {x-version-update;com.azure:azure-data-schemaregistry-avro;current} -->
+  <artifactId>azure-data-schemaregistry-apacheavro</artifactId>
+  <version>1.0.0-beta.6</version> <!-- {x-version-update;com.azure:azure-data-schemaregistry-apacheavro;current} -->
 
-  <name>Microsoft Azure Schema Registry - Avro-specific package for client library</name>
-  <description>Avro-specific package for Azure Schema Registry client library</description>
+  <name>Microsoft Azure client library for serialization using Schema Registry and Apache Avro</name>
```

Comment:
We should also change the name of the serializer classes to include the term Apache.

Reply:
I think this should be worth aligning with .NET because theirs is `SchemaRegistryAvroObjectSerializer`. @JoshLove-msft, @ramya-rao-a  thoughts?

Reply:
Is there a convention already established in Java for other serializers that implement ObjectSerializer?

Reply:
Yes, we have [ApacheAvroSerializer](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core-serializer-avro-apache/src/main/java/com/azure/core/serializer/avro/apache/ApacheAvroSerializer.java) and [JacksonAvroSerializer](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core-serializer-avro-jackson/src/main/java/com/azure/core/serializer/avro/jackson/JacksonAvroSerializer.java) implementations in core.